Remove the REDIS_CONTAINER_NAME environment variable.

plietar opened the pull request:
rrq already respects the REDIS_URL if no host is given. That is more general than just a container name, since it could be running on a different port / outside of Docker / over TLS / require a password.

plietar committed:
Remove the REDIS_CONTAINER_NAME environment variable.

rrq already respects the REDIS_URL if no host is given. That is more
general than just a container name, since it could be running on a
different port / outside of Docker / over TLS / require a password.

M-Kusumgar reviewed (approved):
Thats super useful, did not know that, thanks

plietar committed:
fix flaky test

plietar committed:
Remove rrq_destroy call.

Using finalizers to do this causes some tests to become flaky, as the
queue is destroyed too early. I'm not convinced destroying the queue is
even useful or desirable in production.
